include/seccomp.h.in

```diff
@@ -26,6 +26,7 @@
 #include <inttypes.h>
 #include <asm/unistd.h>
 #include <linux/audit.h>
+#include <linux/seccomp.h>
```

Comment:
What do we need from here?  I'm guessing this is related to the SECCOMP_* macros below?

Reply:
For the definitions of struct seccomp_notif*.

Reply:
Gotcha.  My mistake, I missed the `#ifndef SECCOMP_RET_USER_NOTIF` check in the chunk where you defined everything and I was confused why you needed both :)